qemu/tests/multi_nics_reboot.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+from virttest import error_context
+from virttest import env_process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    1. Boot guest with multi NICs
+    2. Log into a  guest
+    3. Send a reboot command
+    4. Wait until the guest is up again
+    5. Log into the guest to verify it's up again
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    # Get the ethernet cards number from params
+    nics_num = int(params.get("nics_num"))
+    for i in range(nics_num):
+        nics = "nic%s" % i
+        params["nics"] = ' '.join([params["nics"], nics])
+    params["start_vm"] = "yes"
+    env_process.preprocess_vm(test, params, env, params["main_vm"])
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    logging.info("[ %s ] NICs card specified in config file" % nics_num)
+    session = vm.wait_for_login(timeout=int(params.get("login_timeout", 360)))
+    if params.get("reboot_method"):
+        # Reboot the VM
+        for i in range(int(params.get("reboot_count", 1))):
+            session = vm.reboot(session, params["reboot_method"], 0)
+            session.close()
```

Comment:
destroy vm too.

Reply:
I have modified the above three problems,thanks a lot.